### PR TITLE
Add Pipelines product 'Service Quotas' page

### PR DESCRIPTION
**Pull request by mitchbne**
In this PR we introduce a [Service Quotas](https://2774--bk-docs-preview.netlify.app/docs/pipelines/service-quotas) page for limits within the Pipelines product.

CleanShot 2024-09-03 at 14 30 00@2x

### CODE REVIEW

File: data/nav.yml [hunk 1]
```diff
@@ -391,6 +391,8 @@
         - name: "Cluster queue metrics"
           path: "pipelines/cluster-queue-metrics"
           pill: "beta"
+    - name: "Service Quotas"
+      path: "pipelines/service-quotas"
```
Comment (dabarrell) [inline data/nav.yml:395]: I note you've got this under Pipelines, but a handful of the limits are not pipelines-specific. I wonder if we need a global page? or just limit what's on this one to pipelines ones?

Comment (mitchbne) [reply data/nav.yml:395]: Perhaps we do need a global page. At the moment, our navigation has:
- [Pipelines](https://buildkite.com/docs/pipelines)
- [Test Analytics](https://buildkite.com/docs/test-analytics)
- [Packages](https://buildkite.com/docs/packages)
- [APIs](https://buildkite.com/docs/apis)
- [Integrations](https://buildkite.com/docs/integrations)

Are you envisioning a new nav link?

Comment (dabarrell) [reply data/nav.yml:395]: Probably need guidance from @gilesgas on that, but the content on this page atm doesn't really fit under pipelines

Comment (dabarrell) [reply data/nav.yml:395]: Where are the docs for setting up organisations, teams, permissions etc?

Comment (mitchbne) [reply data/nav.yml:395]: > The content on this page atm doesn't really fit under pipelines.

Yeah, you're right. It feels like we're missing a concept of a global/cross-product area

> Where are the docs for setting up organisations, teams, permissions etc?

At the moment those pages sit under the [Pipelines](https://buildkite.com/docs/pipelines) navigation link - https://buildkite.com/docs/team-management/permissions

Comment (dabarrell) [reply data/nav.yml:395]: CleanShot 2024-09-04 at 11 32 35

Oh yeah, these are also definitely not pipelines haha

Comment (mitchbne) [reply data/nav.yml:395]: Nope, they're not 😅😅

File: pages/pipelines/service_quotas.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+Service quotas are put in place to ensure that Buildkite can provide a reliable service to all customers. These quotas are scoped to your organization, and can be increased by contacting support and providing details about your use case.
+
+<table>
+  <tbody>
+    <% [
+      {
+        title: "Invitations per organization",
+        description: "The maximum number of pending invitations for an organization",
+        default_value: "20 invitations"
```
Comment (dabarrell) [inline pages/pipelines/service_quotas.md:18]: Another general thought – how do we keep these values synced with what's in bk/bk?

Comment (dabarrell) [reply pages/pipelines/service_quotas.md:18]: An assertion spec in bk/bk, with a link to this guide doc to update it if an engineer changes the default?

Comment (mitchbne) [reply pages/pipelines/service_quotas.md:18]: We might be able to leverage some internal tooling options (e.g. https://github.com/danger/danger) to help us out here in the bk/bk side 👀

**Comment by gilesgas** [comment]
Hey @dabarrell and @mitchbne ,

Sorry it's taken me a while to revisit this PR, now that we have the new [Platform](https://buildkite.com/docs/platform) area for the docs, I'm wondering if we could publish this work somewhere in here instead?

**Comment by buildkite-systems** [comment]
Preview URL: https://c04f7a71f19f7ec313e853cf5621cba4bb65850a--bk-docs-preview.netlify.app

**Pull request merged**